### DISCUSSION TITLE
Couple of Questions, Comms Update & Ship Fuel Type

### DISCUSSION THREAD

**Discussion by 5th-Domain** [PygmalionOfCyprus/cmo-db-requests#5032] [General]
Good Evening

Just a quick question on if the comms update had been implemented yet, i don't recall any notes on it recently and wondered if id missed something?

Second question i put in a request to change the fuel type of US Navy ships to Diesel as an Update/Correction ticket is that the correct ticket type for such a request? or should it go to tech support?

Thanks for your time.

**Comment by PygmalionOfCyprus** [comment]
We're working on the comms update right now and hope to have it out this year. We implemented all the DB-side schema last year, which would have been the update you read about, but we had to prioritize other things in 2024 dev-side.

And yeah, fuel type stuff is DB-related, so post away in here.

**Reply by 5th-Domain** [reply]
Thanks for the reply, looking forward to seeing the comms update.
